README.md

```diff
@@ -195,20 +185,37 @@ end.
 6. Run `make`. It should say `Looking good!` If one of the sanity checks fails,
    address the complaint and repeat. The various sanity checks are defined in
    `common.mk`.
+   
+   Some checks utilize git. If the repository was cloned using http, you may
+   be asked to authenticate. Using ssh is recommended, but for the purposes of
+   this check the credentials may be left blank.
 
-7. Confirm proper configuration, run the following:
+7. Make sure Docker is running, and not as root. Ubuntu users may need to add 
```

Comment:
I think just this first sentence is sufficient. It's possible the procedures for setting up Docker on Ubuntu will change, and it's easier to maintain if we assume the user will be able to figure out how to do this themselves. Do you agree?

I think pointing to Docker's install documentation is also a reasonable alternative here.